hub/config/src/test/java/uk/gov/ida/hub/config/application/CertificateServiceTest.java

```diff
@@ -288,4 +324,14 @@ public void getAllCertificatesReturnsAllTransactionAndMatchingServiceCertificate
         assertThat(actualCertificateSet).containsAll(expectedCertificateSet);
     }
 
+    private void checkForExpectedLogWarnings(Iterable<String> expectedMessages) {
```

Comment:
This is slight overkill as all of the tests in fact check for exactly one expected message (or don't call the check), but while writing it I was expecting there'd be some cases where we'd need to handle several.  I think the overengineered version will be more resilient against future changes than a simple .get(0) version would be, though, so having written it, I'd prefer to keep it.

Reply:
Yup, LGTM.